service_info/static/css/service-admin.css

```diff
@@ -0,0 +1,4 @@
+td.nowrap {
```

Comment:
So we're changing the `nowrap` class to mean "wrap"?  :-)

Let's provide a longer comment explaining why we would do that...